khmer/khmer_args.py

```diff
@@ -504,8 +504,14 @@ def calculate_graphsize(args, graphtype, multiplier=1.0):
         raise ValueError('unknown graph type: ' + graphtype)
 
     if args.max_memory_usage:
+        # subtract about 30MB for the python interpreter if user is requesting
+        # a large amount of memory that suggests they want to limit the
+        # total amount of RAM used for purposes of not running out of memory
+        max_memory = args.max_memory_usage
+        if max_memory > 1e8:
+            max_memory -= 3e7
```

Comment:
Hmm, I'm not sure about this. I think the behavior is good and necessary, and even though it will involve more code I'd suggest this might be better handled on a script level (with properly documented motivations). I think the library code's responsibility is to give the programmer what she asks for.

Something else to consider would be making `1e8` and `3e7` configurable parameters. I'd suggest `1e9` would be a better default for the first.